src/datanode/src/error.rs

```diff
@@ -101,6 +101,12 @@ pub enum Error {
         source: BoxedError,
     },
 
+    #[snafu(display("Failed to get engine : {}", source))]
+    TableEngine {
```

Comment:
```suggestion
    TableEngineNotFound {
```

And add the name to the error msg.